stdlib/spec/describe_container.mt

```diff
@@ -0,0 +1,17 @@
+defmodule Spec
+  deftype DescribeContainer
+    def initialize(name : String)
+      @name = name      
+    end
+
+    def name; @name; end
+
+    def get_path(current : String)
+      when !describe_stack.empty?
+        return describe_stack.pop.get_path("<(@name)> <(current)>")
```

Comment:
Haven't actually run this yet, but is `pop` the desired behavior here? Seems like it would be modifying the stack when you're just trying to get a value from it.

Looks like we don't have a `.last` method that could replace it anyway. Could be worth adding (along with `.first`)?

Reply:
Yeah, i just though it didn't matter as the program exits when a spec fails. I should do a `first` and `last` still though, for #140.

Reply:
Actually that wouldn't work here anyways if there was nested `describe`s. So i'll figure something else